marlowe-dashboard-client/webpack.config.js

```diff
@@ -1,25 +1,12 @@
-'use strict';
+"use strict";
 
-const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const path = require('path');
+const path = require("path");
 
-const isWebpackDevServer = process.argv.some(a => path.basename(a) === 'webpack-dev-server');
+const isDevelopment = process.env.NODE_ENV === "development";
 
-const isWatch = process.argv.some(a => a === '--watch');
-
-const plugins =
-    isWebpackDevServer || !isWatch ? [] : [
-        function () {
-            this.plugin('done', function (stats) {
-                process.stderr.write(stats.toString('errors-only'));
-            });
-        }
-    ]
-    ;
-
-// source map adds 20Mb to the output!
-const devtool = isWebpackDevServer ? 'eval-source-map' : false;
+const devtool = isDevelopment ? "eval-source-map" : false;
```

Comment:
nothing major really but since you are touching this anyway - how about just moving this to the `exports` block below?
```js
module.exports = {
 devtool = isDevelopment ? "eval-source-map" : false
}
```

Reply:
huh, I thought I already did that - it must have got lost somewhere in all the revisions...